sn_networking/src/bootstrap.rs

```diff
@@ -49,16 +49,24 @@ impl SwarmDriver {
     }
 
     pub(crate) fn trigger_network_discovery(&mut self) {
+        let now = Instant::now();
         // The query is just to trigger the network discovery,
         // hence no need to wait for a result.
-        for addr in &self.network_discovery_candidates {
-            let _ = self
+        for addr in self.network_discovery.candidates() {
+            let query_id = self
```

Comment:
the above comment, shall now be updated as we now insert the query_id ?
or maybe just add one more comment to explain what the `query_id` to be used for?
```rust
        // The query is just to trigger the network discovery,
        // hence no need to wait for a result.
```